pom.xml

```diff
@@ -105,6 +105,11 @@
             <scope>provided</scope>
         </dependency>
         <dependency>
+            <groupId>org.eclipse.aether</groupId>
```

Comment:
yes, without adding this dependency maven was crying in the runtime that he can't find class AuthenticationBuilder.